integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ReceiptPolling.java

```diff
@@ -27,24 +27,31 @@
     private IntegrasjonspunktProperties props;
 
     @Autowired
-    private MessageReceiptRepository messageReceiptRepository;
+    private ConversationRepository conversationRepository;
 
     @Autowired
-    ReceiptStrategyFactory receiptStrategyFactory;
+    ConversationStrategyFactory conversationStrategyFactory;
 
     @Autowired
     DpiReceiptService dpiReceiptService;
 
     // TODO: fjernes etter test!
-    // TODO: fjernes etter test!
     @PostConstruct
     private void addTestData() {
-//        messageReceiptRepository.save(MessageReceipt.of("bb8323b9-1023-4046-b620-63c4f9120b62",
-//                "123", "foo", ServiceIdentifier.EDU));
-//        messageReceiptRepository.save(MessageReceipt.of("bb8323b9-1023-4046-b620-63c4f9120b63",
-//                "456", "bar", ServiceIdentifier.EDU));
-//        messageReceiptRepository.save(MessageReceipt.of("bb8323b9-1023-4046-b620-63c4f9120b64",
-//                "123456", "foobar", ServiceIdentifier.EDU));
+//        MessageReceipt r = MessageReceipt.of(ReceiptStatus.SENT, LocalDateTime.now());
+//        conversationRepository.save(Conversation.of("bb8323b9-1023-4046-b620-63c4f9120b62", "foo", "123",
+//                "fooTitle", ServiceIdentifier.EDU, r));
+//        MessageReceipt ra1 = MessageReceipt.of(ReceiptStatus.SENT, LocalDateTime.now());
+//        MessageReceipt ra2 = MessageReceipt.of(ReceiptStatus.DELIVERED, LocalDateTime.now().plusMinutes(1));
+//        conversationRepository.save(Conversation.of("bb8323b9-1023-4046-b620-63c4f9120b63", "bar", "456",
+//                "barTitle", ServiceIdentifier.EDU, ra1, ra2));
+//        MessageReceipt rb1 = MessageReceipt.of(ReceiptStatus.SENT, LocalDateTime.now());
+//        MessageReceipt rb2 = MessageReceipt.of(ReceiptStatus.DELIVERED, LocalDateTime.now().plusMinutes(1));
+//        MessageReceipt rb3 = MessageReceipt.of(ReceiptStatus.READ, LocalDateTime.now().plusMinutes(5));
+//        Conversation c = Conversation.of("bb8323b9-1023-4046-b620-63c4f9120b64", "foobar",  "123456",
+//                "foobarTitle", ServiceIdentifier.EDU, rb1, rb2, rb3);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)